README.md

```diff
@@ -30,6 +43,14 @@ $ make
 $ sudo make install
 ```
 
+# DataBase
+MongoDB Compass: https://www.mongodb.com/products/tools/compass
+
+GUI DataBase Adres: (mongodb+srv://bakcyl324:Bakcyl768324@paragondatabase.jedczob.mongodb.net/)
```

Comment:
Will it work locally for everyone, or will everyone have to configure something to be able to use datebase?

Reply:
@Jakub324768, please take care of it

Reply:
The only thing you need to provide is the server address, in the case of MongoDB Commpas it is the GUI Address and for C++ it is the second one.

Reply:
Why can't I connect to the database?

![image](https://github.com/BakcylProgramowania/paragon_back/assets/73127307/67854105-f5c8-4eb4-aeec-04fea77957e2)